src/graphql/query/userPreferences.graphql

```diff
@@ -0,0 +1,9 @@
+query userPreferencesQuery {
```

Comment:
This query isn't being used, but we do need to load the preferences. You might be able to include this on the TY page query.